images/ha-config/keepalived/tests/verify_failover_image.sh

```diff
@@ -0,0 +1,90 @@
+#!/bin/bash -e
```

Comment:
How about I add a separate ipfailover integration test. This one is really a docker image verification test, so might be good to keep it around.